src/displayBackend/wayland/Surface.cpp

```diff
@@ -99,7 +113,17 @@ void Surface::draw(FrameBufferPtr frameBuffer,
 
 	wl_surface_commit(mWlSurface);
 
+#ifdef WITH_WAYLAND_PRESENTATION_API
+	auto presentation = mCompositor->getPresentation();
+	if (presentation)
+	{
+		mFeedback = wp_presentation_feedback(presentation, mWlSurface);
```

Comment:
Do you need to keep this pointer? I mean can't local var be used here? Example found on github:
```c++
wp_presentation_feedback_add_listener(::wp_presentation_feedback(presentation_, surface_), &kPresentationFeedbackListener, this);
```

Reply:
It can be null and I assume this may cause a crash.